dev/scripts/start_with_docker.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+##Set Your proxy and cache path here
+HTTP_PROXY='http://10.24.221.169:911'
```

Comment:
it's better to check if the env is defined, otherwise set a default value. you can use VARIABLE="${VARIABLE:-value}", in this way users can define their own env. 

Need to document all the envs.

Reply:
sorry forget this review .it‘s not ready

Reply:
Is it feasible to let the user modify the start_with_docker.sh to modify the proxy, hf_token, cache_path settings? , I think the command line to put these variables directly in the readme is a bit too verbose